_data/versions.yml

```diff
@@ -570,8 +906,8 @@ v2.6:
       url: https://github.com/projectcalico/calicoctl/releases/tag/v1.6.1
       download_url: https://github.com/projectcalico/calicoctl/releases/download/v1.6.1/calicoctl
     calico/node:
-      version: v2.6.2
-      url: https://github.com/projectcalico/calico/releases/tag/v2.6.2
+      version: v2.6.1
```

Comment:
why this change in 2.6.2?

Reply:
@bcreane ^^

Reply:
That shouldn't change. only changes should be to v2.6.5 and v3.0.1, plus a RN-only change to v3.0.0.

Reply:
OK this was an error